docs/en/latest/plugins/clickhouse-logger.md

```diff
@@ -35,7 +35,7 @@ The `clickhouse-logger` Plugin is used to push logs to [ClickHouse](https://clic
 
 | Name          | Type    | Required | Default             | Valid values | Description                                                    |
 |---------------|---------|----------|---------------------|--------------|----------------------------------------------------------------|
-| endpoint_addr | string  | True     |                     |              | ClickHouse endpoint.                                           |
```

Comment:
Also let's keep the old conf and mark it as deprecated like what we have done with `upstream.ip` in https://github.com/apache/apisix/blob/master/docs/en/latest/plugins/mqtt-proxy.md#attributes